edalize/libero.py

```diff
@@ -186,19 +194,39 @@ def tcl_file_filter(self, f):
             return file_types[_file_type] + f.name
         return ""
 
-    def constraint_file_filter(self, f, type="ALL"):
+    def syn_constraint_file_filter(self, f):
+        file_types = {
```

Comment:
I don't really understand what this dict does. Could you explain?

Reply:
Oh, that dict used to supply the path to the constraints files when they were imported, but it's obsolete now that we're always linking. I'll take a stab at cleaning that up.